src/Pages/FeaturesPage/FeaturesPage.scss

```diff
@@ -1,54 +1,126 @@
+@use '../../SASS/keyframes';
+
 .feature-page {
-    background-color: #f7e1e9;
+    background-color: #702880;
     width: 100vw;
     height: 100%;
     font-family: 'Montserrat', sans-serif;
     color: whitesmoke;
     font-weight: 600;
 }
 
+p {
+    color: white;
+}
+
 .feature-tiles {
-    border: 5px blue solid;
     width: 100vw;
     height: 75vh;
     display: grid;
-    grid-template-columns: repeat(2, 1fr);
+    grid-template-columns: 50% 50%;
     align-items: center;
     justify-items: center;
 }
 
-.single-feat-tile {
-    border: 5px red solid;
-    width: 25vw;
-    height: 25vh;
-    padding: 0%;
-    border: 0%;
-    margin: 0%;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    &:hover {
-        cursor: pointer;
-        transform: scale(1.1);
-        border: whitesmoke solid 6px;
-        border-radius: 10%;
-        transition: all 0.3s;
+.bear-image {
+    height: 80%;
+    width: 80%;
+    border-radius: 25px;
+    
+    //  &:hover {
```

Comment:
Lets remember to delete this once were all done with this component. (unused code)